Documentation/project-docs/developer-guide.md

```diff
@@ -246,5 +246,23 @@ If coverage succeeds, the code coverage report will be generated automatically a
 
 Code coverage reports from the continuous integration system are available from the links on the front page of the corefx repo.
 
+### Building tests with .NET Native (Windows only)
+
+.NET Native is a technology that allows compiling IL applications down into a native executable and minimal set of native DLLs, containing all needed functionality from the .NET Framework in native format.  For CoreFX tests, .NET Native support in CoreFX is relatively early, but supported.  
+
+```cmd
+// To run a single project with the .NET Native toolchain, set the appropriate build flags:
+cd src\Microsoft.CSharp\tests
+msbuild /t:BuildAndTest /p:TestTFM=netcore50aot  /p:TestNugetRuntimeId=win10-x64-aot /p:UseDotNetNativeToolchain=true
```

Comment:
It kind of sucks that you have to explicitly pass TestNugetRuntimeId. I think it would be better if we actually set that in dir.props based on the TestTFM, along with the UseDotNetNativeToolchain property. That way we would only need to pass TestTFM which I think is acceptable.